backend/app/api/endpoints/installed_plugins.py

```
@@ -1007,6 +1013,46 @@ def init_plugin_submission(
         ) from exc
 
 
+@router.put("/submissions/{submission_id}/artifact", status_code=204)
+async def upload_plugin_submission_artifact(
+    submission_id: int,
+    request: Request,
+    token: str = Query(..., description="Short-lived plugin upload token"),
+    db: Session = Depends(get_db),
```

Ruff (0.16.3)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)